bin/docker-push

```diff
@@ -5,7 +5,7 @@ set -eu
 . bin/_tag.sh
 
 if [ $# -eq 0 ]; then
-    tag="$(clean_head_root_tag)"
+    tag="$(head_root_tag)"
```

Comment:
docker-push should probably require a tag argument as well to prevent accidental push of dirty tags.